File: 1071-binary-prefix-divisible-by-5/binary-prefix-divisible-by-5.cs
public class Solution {
    public IList<bool> PrefixesDivBy5(int[] nums) {
        List<bool> flags = new List<bool>();
        int rem=0;
        foreach(int i in nums){
            rem = (rem*2+i)%5;
            flags.Add(rem==0);
        }
        return flags;
    }
}

File: 1071-binary-prefix-divisible-by-5/binary-prefix-divisible-by-5.py
class Solution:
    def prefixesDivBy5(self, nums: List[int]) -> List[bool]:
        h=[]
        temp=""
        for i in nums:
            temp+=str(i)
            binary=int(temp,2)
            if int(binary)%5==0:
                h.append(True)
            else:
                h.append(False)
        return h